keyboards/dz60/keymaps/boris_burger/keymap.c

```diff
@@ -0,0 +1,83 @@
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```

Comment:
You can remove this, it's already defined when you include QMK_KEYBOARD_H

Reply:
Slight revision to the above.

Step 1: Remove this line.
Step 2: Replace every sequence of  
`______` (six underscores) with  
`_______` (seven underscores)

Reply:
Good catch, I suppose the next question is, @borisburger was there a specific reason you wanted six underscores vs the default seven?

Reply:
Looks like it came originally from the stephengrier keymap.

https://github.com/qmk/qmk_firmware/blob/ab91e07753720f8114d6c427139a1436e6efa3ce/keyboards/dz60/keymaps/stephengrier/keymap.c#L3

Reply:
If this is just a copy and paste, I'd prefer the default seven spaces, if not I'd like a good explanation as to why.

Reply:
Hello,
thank you for the advice, I will change the keymap accordingly. I am sorry, but I am quite new to the hobby and have a lot to learn. This is the first keyboard I am assembling, it is nothing too special and made out of KPrebublic+kbdfans components, but I love how it sounds and feels:
[Imgur](https://i.imgur.com/Pla5w5s.jpg)

I now realise that I got a bit too much excited about getting the keyboard working that I might have submitted the pull request prematurely. While it mostly works as I hoped, I am fighting with some aspects of the layout, especially regarding the idea of mapping **Shift+Backspace** to **Del**.

I am not quite sure how **Poker II/Pok3r** users live without a separate **Del** key - or are they pressing **Fn+Ctrl+Backspace** to get **Ctrl+End**? It seems a bit too much of a complicated combo for a frequently used thing (or is it just me who uses Ctrl+End all the time?)

I am starting to understand why some 60% keyboards that have an arrow cluster in the corner place a separate **Del** key next to the **Up Arrow**. On full size keyboards I was frequently using combinations like:
- **Shift+Del** in file explorer or in Mozilla Thunderbird to permanently delete file/message instead of putting it to trash
- **Ctrl+Del** to delete the word after the cursor

I am going to change the macros so that:

1. **Fn+Backspace -> Del**
2. **Shift+Backspace -> Shift+Del** (because I do not think Shift+Backspace is really used for anything in text editors)
3. **Ctrl+Backspace -> Ctrl+Del** (because while Ctrl+Backspace is an actual shortcut for deleting the word to the left of the cursor in text editors, I did not even know that this shortcut existed until today).

Please give me some time to revise the layout, and I am sorry to have not have been respectful to your time and submitting the pull request prematurely.